src/d3a/models/area/__init__.py

```diff
@@ -313,7 +313,7 @@ def next_market(self):
 
     @property
     def current_market(self):
-        """Returns the 'current' market (i.e. the one currently 'running')"""
+        """Returns the 'last' market (i.e. the one that has be finished last)"""
```

Comment:
To me personally only word `last` confuses me, method `current_market` also doesn't explains what would be returned. I would vouch to be a bit more verbose here, for instance rename the method `current_market` to something like `most_recent_past_market`

Reply:
Although i agree that current_market is not the right wording, I dont want to rename the function. I just wanted to adapt the doc string because this is a copy paste from "next_market".
I will change it to:
"""Returns the 'most recent past market' market (i.e. the one that has been finished last)""" 
OK with you @faizan2590 ?

Reply:
But I think it is trivial to rename the method `current_market` --> `most_recent_past_market`. Or if you think it needs to be done more carefully, please create a small sub task for future.

Reply:
@spyrostz do you agree with the renaming? If so, I will add it in this PR.

Reply:
Frankly, I am by now used to the term current_market. And it kind of makes sense to my eyes, because all markets refer to the future, thus the current market (last past market) refers to the current point in time. I predict that even if we rename it to last_past_market, we are going to have the opposite problem, meaning that one might expect that the last past market is not the current time, but one slot before that.
Despite that, if you both agree that the renamed version looks better feel free to do this.